cmake/AddPdfTarget.cmake

```diff
@@ -43,7 +43,7 @@ function (add_pdf_target docname lang entities figures dtd_files)
         OUTPUT "${BUILD_DIR}/${fofile}"
         COMMAND ${XSLTPROC} ${XSLTPROCFLAGS} ${XSLTPROCFLAGS_FO}
                             -o "${BUILD_DIR}/${fofile}"
-                            --stringparam fop1.extensions 1
+                            --stringparam gnc.lang ${lang}
```

Comment:
Ok, though the  cmake bit that defined  the papersize in the past should only be removed as part of this commit.